src/Item/ItemRegistry.cs

```diff
@@ -0,0 +1,15 @@
+﻿
+using Vintagestory.API.Common;
+
+namespace RPVoiceChat
+{
+    public class ItemRegistry
+    {
+        public static void RegisterItems(ICoreAPI api)
+        {
+            api.RegisterItemClass("handbell", typeof(HandbellItem));
+            api.RegisterItemClass("handheldradio", typeof(RadioItem));
+            api.RegisterItemClass("telegraphwire", typeof(TelegraphWireItem));
```

Comment:
Do these two need really empty item classes to work? Or is it just a setup for future?

Reply:
Setup for the future